packages/node_modules/@webex/webex-core/src/interceptors/redirect.js

```diff
@@ -74,6 +74,8 @@ export default class RedirectInterceptor extends Interceptor {
       response.body.errorCode === LOCUS_REDIRECT_ERROR &&
       response.body.location) {
       options = clone(options);
+
+      this.webex.logger.warn('redirect: url redirects needed from', options.url);
```

Comment:
this and the other change in this file seem unrelated to the roap issue. This is for the locus redirect issue, isn't it? So just checking if it's here intentionally.
Also, these new logs are logging `options.url` while the code around here is using `options.uri` so just double checking that you want to log `url` and not `uri`